### PR TITLE
Update changelog for 2.10.1

### PR DESCRIPTION
**Pull request by demartinofra**
Signed-off-by: Francesco De Martino <fdm@amazon.com>

**Please See** [Git Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions)

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
   - At the time of this version launch:
     - AWS Lustre and ARM instance type are not supported in me-south-1, af-south-1 and eu-south-1  
     - AWS Batch is not supported in af-south-1
-    - EBS io2 is not supported in af-south-1 and eu-south-1 
-- Remove CloudFormation DescribeStacks API call from AWS Batch Docker entrypoint. This removes the possibility of job
+    - EBS io2 is not supported in af-south-1 and eu-south-1
+- Install Arm Performance Library 20.2.1 on ARM AMI(CentOS8, Alinux2, Ubuntu1804)
```
Comment (enrico-usai) [inline CHANGELOG.md:14]: `Install Arm Performance Libraries 20.2.1 on ARM AMIs (CentOS8, Alinux2, Ubuntu1804)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
...
-- Enable support for p4d as head node instance type. (p4d was already supported as compute node in 2.10.0)
 - Upgrade EFA installer to version 1.11.0
   - EFA configuration: ``efa-config-1.6`` (from efa-config-1.5)
   - EFA profile: ``efa-profile-1.2`` (from efa-profile-1.1)
   - EFA kernel module: ``efa-1.10.2`` (no change)
   - RDMA core: ``rdma-core-31.2amzn`` (from rdma-core-31.amzn0)
   - Libfabric: ``libfabric-1.11.1amzn1.0`` (from libfabric-1.11.1amzn1.1)
   - Open MPI: ``openmpi40-aws-4.0.5`` (no change)
+- Upgrade Intel MPI to version U8.
+- Upgrade NICE DCV to version 2020.2-9662
```
Comment (enrico-usai) [inline CHANGELOG.md:38]: minor: missing final period.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
...
+- Set default systemd runlevel to multi-user.target on all OSes during ParallelCluster official ami creation.
+  The runlevel is set to graphical.target on head node only when DCV is enabled. This prevents the execution of
+  graphical services, such as x/gdm, when they are not required.
+- Download Intel MPI and HPC packages from S3 rather than Intel yum repos.
+- Change the default of instance types from the hardcoded `t2.micro` to the free tier instance type 
+    (`t2.micro` or `t3.micro` dependent on region). In regions without free tier, the default is `t3.micro`.
+- Enable support for p4d as head node instance type. (p4d was already supported as compute node in 2.10.0)
```
Comment (enrico-usai) [inline CHANGELOG.md:45]: minor: let's move the period at the end of the sentence.

 in 2.10.0 --> since ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
   - At the time of this version launch:
     - AWS Lustre and ARM instance type are not supported in me-south-1, af-south-1 and eu-south-1  
     - AWS Batch is not supported in af-south-1
-    - EBS io2 is not supported in af-south-1 and eu-south-1 
-- Remove CloudFormation DescribeStacks API call from AWS Batch Docker entrypoint. This removes the possibility of job
+    - EBS io2 is not supported in af-south-1 and eu-south-1
+- Install Arm Performance Library 20.2.1 on ARM AMI(CentOS8, Alinux2, Ubuntu1804)
+- Install EFA kernel module on ARM instances with `alinux2` and `ubuntu1804`. This enables support for `c6gn` instances.
+- Add support for io2 and gp3 EBS volume type.
+- Remove CloudFormation DescribeStacks API call from AWS Batch Docker entrypoint. This removes the risk of job
```
Comment (mauri-melato) [inline CHANGELOG.md:17]: Move to Bug Fixes section

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
...
   failures due to CloudFormation throttling.
-- Install EFA kernel module also on ARM instances with `alinux2` and `ubuntu1804`
 - Add `iam_lambda_role` parameter under `cluster` section to enable the possibility to specify an existing IAM role to 
   be used by AWS Lambda functions in CloudFormation. 
   When using `sge`, `torque`, or `slurm` as the scheduler, 
   `pcluster` will not create any IAM role if both `ec2_iam_role` and `iam_lambda_role` are provided.
-- Add support for io2 and gp3 EBS volume type.
+- Improve error handling in slurm plugin processes when clustermgtd is down.
```
Comment (mauri-melato) [inline CHANGELOG.md:23]: what about, Improve robustness of a Slurm cluster when clustermgtd is down ?

File: CHANGELOG.md [hunk 6]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
...
+- Configure NFS threads to be max(8, num_cores) for performance. This enhancement will not take effect on Ubuntu 16.04
+  unless the instance is rebooted.
```
Comment (mauri-melato) [inline CHANGELOG.md:25]: I'd avoid the 'unless' part.

File: CHANGELOG.md [hunk 7]
```diff
@@ -10,35 +10,51 @@ CHANGELOG
...
+- Set default systemd runlevel to multi-user.target on all OSes during ParallelCluster official ami creation.
+  The runlevel is set to graphical.target on head node only when DCV is enabled. This prevents the execution of
+  graphical services, such as x/gdm, when they are not required.
+- Download Intel MPI and HPC packages from S3 rather than Intel yum repos.
+- Change the default of instance types from the hardcoded `t2.micro` to the free tier instance type 
+    (`t2.micro` or `t3.micro` dependent on region). In regions without free tier, the default is `t3.micro`.
+- Enable support for p4d as head node instance type. (p4d was already supported as compute node in 2.10.0)
+- Pull Amazon Linux Docker images from public ECR when building docker image for `awsbatch` scheduler.
+- Increase max retry attempts when registering Slurm nodes in Route53.
+- Use inclusive language in user facing messages and internal naming convention.
```
Comment (mauri-melato) [inline CHANGELOG.md:48]: Please remove.

**Review by mauri-melato — COMMENTED** [review]
Just few comments then looks LGTM, thank you!

**Review by enrico-usai — APPROVED** [review]

**Review by enrico-usai — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -10,35 +10,49 @@ CHANGELOG
...
+- Upgrade EFA installer to version 1.11.0.
   - EFA configuration: ``efa-config-1.6`` (from efa-config-1.5)
   - EFA profile: ``efa-profile-1.2`` (from efa-profile-1.1)
   - EFA kernel module: ``efa-1.10.2`` (no change)
   - RDMA core: ``rdma-core-31.2amzn`` (from rdma-core-31.amzn0)
   - Libfabric: ``libfabric-1.11.1amzn1.0`` (from libfabric-1.11.1amzn1.1)
   - Open MPI: ``openmpi40-aws-4.0.5`` (no change)
+- Upgrade Intel MPI to version U8.
+- Upgrade NICE DCV to version 2020.2-9662.
+- Set default systemd runlevel to multi-user.target on all OSes during ParallelCluster official ami creation.
```
Comment (mauri-melato) [inline CHANGELOG.md:36]: uppercase AMI

**Review by mauri-melato — APPROVED** [review]

**Pull request merged**